app/models/genre.rb

```diff
@@ -15,4 +15,12 @@ class Genre < ApplicationRecord
   validates :name, presence: true, uniqueness: true
 
   normalized_col_generator :name
+
+  def merge(other)
+    tracks = other.tracks
+    tracks.find_each do |track|
```

Comment:
```suggestion
    other.tracks.find_each do |track|
```
You should avoid shadowing a method name.